// bot-help/help.js
exports.help = { 
    embed: {
        color: 3447003,
        author: {
            name: "Available Commands",
        },
        description: "This is the help section, commands are not case sensitive.\n ahoy help (command) for syntax.",
        fields: [{
                name: "add",
                value: "Give a member activity points."
            },
            {
                name: "remove",
                value: "Take activity points from a member."
            },
            {
                name: "list",
                value: "List all members along with activity points."
            },
            {
                name: "find",
                value: "Show activity points for a specific member."
            },
            {
                name: "payout",
                value: "Sends back payment tiers for members and hard resets activity points."
            }
        ],
        timestamp: new Date(),
        footer: {
            text: "Mutiny"
        }
    }
}

exports.test = function(){
    console.log("test");
}
